app/views/users/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<div>
+  <% @users.each do |user| %>
+    <%= user.name %>
+  <% end %>
+</div>
```

Comment:
- [ ] Same here, Please add a placeholder written in plain HTML like: `Here is a list of all users`